src/backend/vulkan/src/device.rs

```diff
@@ -711,12 +612,34 @@ impl d::Device<B> for Device {
 
         let framebuffer = unsafe {
             self.raw.0.create_framebuffer(&info, None)
-                        .expect("error on framebuffer creation")
-        };
+        }.expect("error on framebuffer creation");
 
         n::FrameBuffer { raw: framebuffer }
     }
 
+    fn create_shader_module(&mut self, spirv_data: &[u8]) -> Result<n::ShaderModule, d::ShaderError> {
+        assert_eq!(spirv_data.len() & 3, 0);
```

Comment:
Could we add a comment for this. The size of SPIR-V modules requiring to be a multiple of the size of DWORD is not that obvious imo.